code/create/create-dta.sh

```diff
@@ -0,0 +1,46 @@
+# First argument: Path to install the DTA_study data
+# Second argument: which sourcedata to use
+# Example-call for installing DTA_sourcedata_reduced:
+# >> ./create_BIDS_DTA.sh . /data/BnB_TEMP/Kadelka/DTA_sourcedata_reduced/
+
+# create a dataset
+datalad create -c hirni $1
+cd $1
+
+# installing the necessary code into the dataset
+datalad install -d . -s git@github.com:TobiasKadelka/build_dta.git code/build_dta
+
+# modify the rules for the spec, so the dta-rules will be used
+git config -f .datalad/config --add datalad.hirni.dicom2spec.rules code/build_dta/code/create/dta_dicom2spec_rules.py
+
+# configuring the .datalad/config, so it knows where to find the procedure for renaming the fieldmaps
+# git config -f .datalad/config --add datalad.locations.dataset-procedures code/build_dta/code/procedures
+
+# THIS IS NOT NECESSARY, WHEN FILES ARE ALREADY IN TARS ON CLUSTER
+# get the source_data for DTA1
+# ./code/tmp/tar-DTA1_output.sh
+# source_data for DTA2
+# ./code/tmp/tar-DTA2_output.sh
+
```

Comment:
The outcommented block above seems outdated, since at this point those paths don't exist (code/tmp/..). Adjust or delete that block. Remember: Two years from now, there might be a need to figure out, what exactly was going on. Even if outcommented, invalid left-overs can only cause confusion then.